aikit/models/ordinal.py

```diff
@@ -0,0 +1,364 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Mar 24 16:32:08 2020
+
+@author: LionelMassoulard
+"""
+
+import numpy as np
+import pandas as pd
+
+from sklearn.base import BaseEstimator, ClassifierMixin, RegressorMixin, is_classifier, is_regressor
+from sklearn.preprocessing import OrdinalEncoder, KBinsDiscretizer
+
+
+from aikit.tools.data_structure_helper import make2dimensions, make1dimension, convert_generic
+from aikit.enums import DataTypes
+from aikit.transformers.categories import _OrdinalOneHotEncoder
+
+
+
+class _BaseOrdinalClassifier(BaseEstimator, ClassifierMixin):
+    
+    def _prepare_target(self, y, klass, conversion_type):
```

Comment:
maybe add some docstring about the intent of the method, what it stores in the class state etc...

Reply:
done